Mod2/Project/index.md

```diff
@@ -0,0 +1,76 @@
+# Message Logger
+
+## Learning Goals
+- Demonstrate:
+    * Version Control, including branches and pull requests.
+    * Understanding of database table relationships.
+    * Database CRUD with an ORM.
+
+- Explore:
+    * Dive into a new (not written by you) code base!
+    * Using TDD to add additional features!
```

Comment:
I think it might be helpful to be more explicit about when and what tests we expect from them as part of this project.

Reply:
For sure!